ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -957,6 +957,48 @@ def test_torch_sqrt(
     )
 
 
+# pow
+@handle_cmd_line_args
+@given(
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=tuple(
+            set(ivy_np.valid_float_dtypes).intersection(
+                set(ivy_torch.valid_float_dtypes)
+            )
+        ),
+        num_arrays=2,
+        min_value=-1e04,
+        max_value=1e04,
+        allow_inf=False,
+    ),
+    num_positional_args=helpers.num_positional_args(
+        fn_name="functional.frontends.torch.pow"
+    ),
+)
+def test_torch_pow(
+    dtype_and_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fw,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        fw=fw,
+        frontend="torch",
+        fn_tree="pow",
+        input=np.asarray(x[0], dtype=input_dtype[0]),
+        other=np.asarray(x[1], dtype=input_dtype[1]),
```

Comment:
This frontend test is incorrect. Please check the arguments that are being to the function again! As `pow` doesn't have an `other` argument from your function definition.

Reply:
You're right, I must have messed that up when resolving the merge conflicts, thank you for spotting that. Re-requesting review, thanks in advance